packages/block-editor/src/components/page-template-picker/picker.js

```diff
@@ -2,6 +2,8 @@
  * WordPress dependencies
  */
 import { useState } from '@wordpress/element';
+import { parse } from '@wordpress/blocks';
+import { withDispatch } from '@wordpress/data';
```

Comment:
Since this is a functional component, you can use `useDispatch` instead:

```jsx
const { editPost, resetEditorBlocks } = useDispatch( 'core/editor' );
```